lib/dfp_helper.rb

```diff
@@ -15,11 +15,12 @@ def dfp_helper_slot(_i, options = {})
       _id = options[:div_id]
       _id ||= "div-gpt-ad-#{@@dfp_helper_id}-#{dfp_helper_slots.size}"
       _size = options[:size] || _i.match(/\d+x\d+/)[0].split('x')
+      _styles = options[:responsive_mapping] ? '' : "width:#{_size[0]}px; height:#{_size[1]}px;"
```

Comment:
alright -- sounds fine but typically you only name variables with an underscore if they are unused parameters in a method... e.g. `_size` should probably be `size` since it's passed into your `_styles` assignment... Anyway, not a big deal at all no need to change it was just curious